examples/panel-basic/src/components/SimplePanel/SimplePanel.tsx

```diff
@@ -4,10 +4,10 @@ import { TimeSeries, TooltipPlugin, TooltipDisplayMode, ZoomPlugin } from '@graf
 import { SimpleOptions } from '../../types';
 import { testIds } from '../testIds';
 
-interface Props extends PanelProps<SimpleOptions> {}
+interface Props extends PanelProps<SimpleOptions> {} /** Entry point */
 
-export function SimplePanel({
-  options,
+export function SimplePanel({ /** Takes in a list of props used in this example */
+  options, /** Options declared within module.ts, allow creation of form elements on Grafana sidebar */
```

Comment:
use `//` for same line comments (except inside JSX as you did below)

Reply:
It's not only the options declared in the `module.ts` but also standard options set by Grafana.

If you open the panel edit sidebar there are a set of standard options that are displayed for all panels (if they are not explicit disabled by the plugin). They will also be passed in the options.

I think we probably should cover these props in some other place and just reference that documentation from this example.

Reply:
I have changed this comment to "// Options declared within module.ts and standard Grafana options". I'm not sure this is all that helpful, perhaps we can leave it out? If there's a discussion of options elsewhere we can link to, we can add a link to that in a later PR.